css/style.css

```diff
@@ -0,0 +1,293 @@
+body {
+    color: #000000;
+    font-family: 'montserrat-normal', sans-serif;
+}
+
+.title,
+.bag__title-bold,
+.bag__title,
+.bag-title-bold,
+.bag-title,
+.name {
+    font-family: 'montserrat-bold', sans-serif;
+}
+
+/* 1 section*/
+
+.new-collection__background {
+    position: absolute;
+    max-width: 461px;
+    max-height: 900px;
+}
+
+.new-сollection__header,
+.new-сollection__main,
+.new-сollection__menu,
+.new-сollection__menu {
+    position: relative;
+}
+
+.logo {
+    margin-right: 238px;
+    max-width: 98px;
+    max-height: 37px;
+    margin-top: 40px;
+    margin-left: 52px;
+}
+
+.logo-mobil {
+    position: absolute;
+    width: 98px;
+    height: 37px;
+    padding: 5% 0 0 20%; 
+}
+
+.lang {
+    margin-top: 42px;
+}
+
+.new-сollection__menu {
+    width: 92px;
+    height: 92px;
+    position: absolute;
+    top: 0;
+    right: 0;
+ }
+
+ .new-сollection__menu-box {
+    opacity: 0.05;
+    background: #000;
+    width: 92px;
+    height: 92px;
+ }
+
+ .menu {
+     position: absolute;
+     top: 40%;
+     right: 30%; 
+ }
+
+ .girl {
+    max-width: 855.54px;
+    height: 839px;
+    left: 49.28px;
+    top: 60.92px;
+    margin-left: 50px;
+    background: url(../img/girl.png) no-repeat center top 100px;
+ }
+
+.title {
+    font-style: normal;
+    font-weight: bold;
+    font-size: 80px;
+    line-height: 82px;
+    position: absolute;
+    width: 475px;
+    height: 164px;
+    left: 789px;
+    top: 210px;
+    margin: 0;
+}
+
+.text {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 21px;
+    color: #000000;
+    position: absolute;
+    width: 467px;
+    height: 127px;
+    left: 789px;
+    top: 372px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5388782)